app/src/main/java/com/quarkworks/apartmentgroceries/grocery/GroceryCardPagerFragment.java

```diff
@@ -0,0 +1,84 @@
+package com.quarkworks.apartmentgroceries.grocery;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.quarkworks.apartmentgroceries.R;
+import com.quarkworks.apartmentgroceries.service.DataStore;
+import com.quarkworks.apartmentgroceries.service.models.RGroceryItem;
+import com.quarkworks.apartmentgroceries.user.UserDetailActivity;
+
+/**
+ * Created by zz on 10/22/15.
+ */
+public class GroceryCardPagerFragment extends Fragment {
+    private static final String TAG = GroceryCardPagerFragment.class.getSimpleName();
+
+    private static final String GROCERY_ID = "groceryId";
+    private String groceryId;
+    private RGroceryItem rGroceryItem;
+
+    static GroceryCardPagerFragment newInstance(String groceryId) {
+        GroceryCardPagerFragment groceryCardPagerFragment = new GroceryCardPagerFragment();
+        Bundle args = new Bundle();
+        args.putString(GROCERY_ID, groceryId);
+        groceryCardPagerFragment.setArguments(args);
+
+        return groceryCardPagerFragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        groceryId = getArguments().getString(GROCERY_ID);
+        rGroceryItem = DataStore.getInstance().getRealm().where(RGroceryItem.class)
+                .equalTo(GROCERY_ID, groceryId).findFirst();
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+
+        /**
+         * Get view references
+         */
+        View rootView = inflater.inflate(R.layout.grocery_card_pager_fragment, container, false);
```

Comment:
Personally I make the inflation the very first line of the method, like this:

    View rootView = inflater.inflate(R.layout.grocery_card_pager_fragment, container, false);

    /**
     * Get view references
     */
    //etc.